packages/gatsby-plugin-preact/package.json

```diff
@@ -25,7 +27,8 @@
   "main": "index.js",
   "peerDependencies": {
     "gatsby": "^2.0.0",
-    "preact": "^10.3.4"
+    "preact": "^10.3.4",
+    "preact-render-to-string": "^5.1.8"
```

Comment:
Actually - adding new peer dep warrants major version bump? (otherwise users updating within same major will have broken setups)

Reply:
hmm - where is this used?

Reply:
yes, it's used for ssr now :) seems like our alias wasn't setup correctly. I'll push a  major